proj/src/main/webapp/script.js

```diff
@@ -50,17 +50,100 @@ function errorCallback(err){
 function getLocation() {
     if(navigator.geolocation){
         navigator.geolocation.getCurrentPosition(successCallback,errorCallback,{timeout:10000, enableHighAccuracy:false});
-    }
-    else{
+    } else{
         console.log("Error. Geolocation not supported or not enabled");
     }
     return;
 }
 
+/*
+    Setting password 
+*/ 
+
+// Check if the password meets the password requirement with every character
+$('#pass').on('keyup', function(){
+    meetRequirements();
+    pwLength.hidden = false;
+    pwNumber.hidden = false;
+    pwSymbol.hidden = false;
+});
+
+// Check if the password is strong enough 
+function meetRequirements(){
+    const password = document.getElementById("pass");
+    var goodPassword = false;
+    passwordLength(password);
+    containsNumber(password);
+    containsSymbol(password);
+
+    /* returns true or false so that the sign up button is enables with a 
+    strong password and matching confirmation password **/
+    var goodPassword = [passwordLength,containsNumber,containsSymbol].every(function(handler){return handler(password)})
+    if (goodPassword){
+        return true;
+    } else{
+        return false;
+    }
```

Comment:
I think you can just `return goodPassword;` and save yourself 4 lines of code.

Reply:
gotcha, it's changed

Reply:
I don't think i see the change.

Reply:
okay i pushed my repo again